README.md

```diff
@@ -237,3 +237,8 @@ If you want to grant members of other groups ability to view the results, you ca
 names in the django settings using the `XBLOCK_POLL_EXTRA_VIEW_GROUPS` setting, for example:
 
     XBLOCK_POLL_EXTRA_VIEW_GROUPS = ['poll_staff']
+
+## Adding Translated Strings to your XBlock
+
+You can follow the steps as defined here.
+[Xblock-translations]http://edx.readthedocs.io/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html#adding-translated-strings-to-your-xblock
```

Comment:
@asadiqbal08 This link does not display correctly in the rendered Markdown. You'll need to wrap the actual link in parentheses.